src/main/java/graphql/schema/GraphQLList.java

```diff
@@ -31,52 +38,66 @@ public static GraphQLList list(GraphQLType wrappedType) {
         return new GraphQLList(wrappedType);
     }
 
-    private GraphQLType wrappedType;
 
     public GraphQLList(GraphQLType wrappedType) {
         assertNotNull(wrappedType, "wrappedType can't be null");
-        this.wrappedType = wrappedType;
+        this.originalWrappedType = wrappedType;
     }
 
 
     @Override
     public GraphQLType getWrappedType() {
-        return wrappedType;
+        if (replacedWrappedType != null) {
+            return replacedWrappedType;
+        }
+        return originalWrappedType;
```

Comment:
a use case for `? :` operator or a helper method

`<T> T orElse(T one, T two)`

Reply:
done